apps/cloud_composer/include/pcl/apps/cloud_composer/tools/impl/organized_segmentation.hpp

```diff
@@ -174,8 +173,4 @@ pcl::cloud_composer::OrganizedSegmentationTool::performTemplatedAction (const QL
 }
 
 
-#define PCL_INSTANTIATE_performTemplatedAction(T) template void pcl::cloud_composer::OrganizedSegmentationTool::performTemplatedAction<T> (QList <const CloudComposerItem*>);
-
-
-
-#endif //IMPL_TRANSFORM_CLOUDS_HPP_
+#define PCL_INSTANTIATE_performTemplatedAction(T) template void pcl::cloud_composer::OrganizedSegmentationTool::performTemplatedAction<T> (QList <const CloudComposerItem*>);
```

Comment:
New line at the end of the file

Reply:
Are you saying the file need an empty line at the end? The line at the end was originally there